understanding/22/pointer-target-spacing.html

```diff
@@ -0,0 +1,153 @@
+      
+        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size.</p>
+
+        <figure id="pointer-target-44x44">
+            <img src="img/pointer-target-example1.png" alt="A series of icons with measures showing they are each 44 by 44 pixels wide and high." />
+            <figcaption>The dimension of each icon target in a row is 44 by 44 CSS pixels, so the targets pass this Success Criterion without additional spacing.</figcaption>
+        </figure>
+
+        <figure id="pointer-target-38x38-spacing">
+            <img src="img/pointer-target-example2.png" alt="A row of icons with measures showing they are each 38 by 38 pixels wide and high, with 6 pixel spacing between them." />
+            <figcaption>The dimension of each icon target in a row is 38 by 38 CSS pixels, so a spacing of 6 CSS pixels is required to arrive at 44 CSS pixels for the combinded width and spacing of the target. No adjacent targets are present above and below, i.e., there is sufficient vertical spacing for the height of 38 CSS pixels to be sufficient.</figcaption>
+        </figure>
+
+        <figure id="pointer-target-44x20-spacing">
+            <img src="img/pointer-target-example3.png" alt="A row of buttons which are more that 44px wide and 20px high. There are no targets above or below." />
+            <figcaption>The widths of the button with adjacent neighbors is at least 44 CSS pixels, while the height is only 22 CSS pixels. The target height combined with the vertical spacing above and below exceeds 44 CSS pixels, therefore, the targets pass this Success Criterion.</figcaption>
+        </figure>
+      
+        <figure id="pointer-target-44-20-nospace">
+            <img src="img/pointer-target-example4.png" alt="A row of icons which are 44px wide and 20px high, with other targets above and below with only 2px spacing." />   
+            <figcaption><strong>Fail:</strong> Two rows of buttons, all with a height of only 20 CSS pixels. In the top row, buttons all have a width of more than 44 CSS pixels. For the top row, targets above and below are in close proximity. The top spacing of the first row is 4 px while the bottom spacing is 2 px, which adds up to 26 px for target height and vertical spacing combined. Therefore, the buttons in the top row fail the Success Criterion. The button "Pop" in the second row is only 32px wide with no horizontal spacing, so it also fails the Success Criterion even when there is sufficient space below to meet the vertical spacing requirement.</figcaption>
+        </figure>
+
+        <p>Where targets are embedded inside other targets, the inner target either needs a size of at least 44 x 44 CSS pixels or must placed at the edge or corner in a way that sufficient adjacent spacing is available in each dimension.</p>
```

Comment:
"or must placed at the edge" > "or must be placed at the edge"

Reply:
perhaps slightly awkward wording "in a way that sufficient adjacent spacing is available in each dimension".

maybe "to make sure that there is sufficient adjacent spacing in each dimension"